reference/twig_reference.rst

```diff
@@ -721,10 +715,6 @@ The available attributes are:
 * ``app.debug``
 * ``app.security``
 
-.. versionadded:: 2.6
-     The ``app.security`` global is deprecated as of 2.6. The user is already available
-     as ``app.user`` and ``is_granted()`` is registered as function.
```

Comment:
Removing this one is wrong. The explanation saying it is deprecated is necessary on 2.8